dark/aa.py

```diff
@@ -1223,14 +1219,14 @@ def _Match(a, b):
                     read2GapOffsets.append(offset)
                 else:
                     # Neither is a gap character.
-                    if _Match(a, b):
-                        MatchCount += 1
+                    if a == b:
```

Comment:
Just BTW, you could also do `matchCount += a == b`.  The comparison is in a numeric context, so Python converts the `True` / `False` to an integer `1` or `0` and everyone lives happily ever after. At least until someone who doesn't know Python that well tries to read your code.....   That situation is one of the rare places I'd use a pair of not-strictly-needed parens, to improve readability: `matchCount += (a == b)`. See also http://blogs.fluidinfo.com/terry/2012/12/18/omit-needless-parens/

Reply:
Ok, will leave it explicit